### PR TITLE
Update communication readme link to match new location.

### PR DESCRIPTION
**Pull request by loicbourgois**
**What this PR does / why we need it**:
Update communication readme link to match new location.
[Before](https://github.com/kubernetes/community/blob/master/communication.md)
[After](https://github.com/kubernetes/community/blob/master/communication)

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
None

**Special notes for your reviewer**:
None

**Release note**:
```release-note
NONE
```

### CODE REVIEW

**Comment by loicbourgois** [comment]
Signed CLA

File: README.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ That said, if you have questions, reach out to us
 [announcement]: https://cncf.io/news/announcement/2015/07/new-cloud-native-computing-foundation-drive-alignment-among-container
 [Borg]: https://research.google.com/pubs/pub43438.html
 [CNCF]: https://www.cncf.io/about
-[communication]: https://github.com/kubernetes/community/blob/master/communication.md
+[communication]: https://github.com/kubernetes/community/blob/master/communication
```
Comment (cblecker) [inline README.md]: Please replace this link with: `https://git.k8s.io/community/communication`

**Comment by loicbourgois** [comment]
@cblecker Should other urls inf `https://github.com/kubernetes` be updated to use `https://git.k8s.io/`
If so, is it appropriate to do it in this PR ?

**Comment by cblecker** [comment]
@loicbourgois if you want to run through all of the links in this top level README, yes that would be helpful.

**Comment by loicbourgois** [comment]
@cblecker Should be ok for merge

**Pull request merged**